File: PrevozMVC/ePrevoz.WEB/Models/PorukeVM.cs
using ePrevoz.DATA.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ePrevoz.WEB.Models
{
    public class PorukeVM
    {

       
        public int Id { get; set; }
     

        public string Text { get; set; }

        public byte[] Data { get; set; }

        public string DataType { get; set; }

     
        public int KorisnikPoslaoId { get; set; }
        public int KorisnikPrimioId { get; set; }

        public DateTime DatumKreiranja { get; set; }
        public int Status { get; set; }
    }
}

File: PrevozApp/Prevoz/app/src/main/java/com/basic/prevoz/Models/PorukeVM.java
package com.basic.prevoz.Models;

import java.io.Serializable;
import java.util.Date;

/**
 * Created by Developer on 27.06.2017..
 */

public class PorukeVM implements Serializable {


    public  int Id;
    public String Text;
    public int KorisnikPoslaoId;
    public int KorisnikPrimioId;

    public Date DatumKreiranja;
    public int Status;

    public  KorisniciVM Korisnik;
    public String DataType;
}
